### PR TITLE
Set nodetool parameters

### PR DESCRIPTION
**Pull request by bhagenbourger**
Aims to solve #34

### CODE REVIEW

File: medusa/cassandra_utils.py [hunk 1]
```diff
@@ -483,7 +504,7 @@ def is_node_up(config, host):
     if int(config.cassandra.is_ccm) == 1:
         args = ['ccm', 'node1', 'nodetool']
     else:
-        args = ['nodetool', '-h', host]
+        args = Nodetool(config.cassandra).nodetool + ['-h', host]
```
Comment (arodrime) [inline medusa/cassandra_utils.py]: Is there a reason not to use the standard `self._nodetool.nodetool` here? (like for lines above) as we have:

```
self._nodetool = Nodetool(cassandra_config)
```

**Comment by arodrime** [comment]
I don't have a cluster around but integration tests are passing. I'm ok to merge this once the small thing above is fixed.

```
1 feature passed, 0 failed, 0 skipped
14 scenarios passed, 0 failed, 0 skipped
251 steps passed, 0 failed, 0 skipped, 0 undefined
Took 7m53.828s
```

File: medusa/cassandra_utils.py (hunk 1 above)
Comment (bhagenbourger) [reply medusa/cassandra_utils.py]: The method `is_node_up` is not member of `Cassandra` class. So I can't access to `self` here

Comment (arodrime) [reply medusa/cassandra_utils.py]: oh! I remember that. We never fixed it... It should be part of Cassandra, but then for now this is probably good enough.

Comment (bhagenbourger) [reply medusa/cassandra_utils.py]: ok, thanks for the review @arodrime

**Comment by jfharden** [comment]
@arodrime Is this able to get merged? The issue was marked as resolved and we could really use this feature (we have the same problem with needing authentication credentials for nodetool)

**Comment by arodrime** [comment]
@jfharden hello. I'm back from the winter break. Sorry about the delay there. I wanted to test it on a cluster and had some troubles running my test environment 🙃.

For what it's worth you can already install this version if you are in a hurry (or just willing to give this a try).

To install this version, you can run this on all nodes:

```
sudo apt-get install -y python3-pip && sudo pip3 install git+https://github.com/bhagenbourger/cassandra-medusa.git@set_nodetool_parameters --upgrade --force-reinstall 
```

**Comment by jfharden** [comment]
@arodrime Thanks, I tried this branch and have it working with jmx authentication. Thanks @bhagenbourger

**Comment by arodrime** [comment]
Let's merge this, if you tested it. It worked for me too (even though I did not have much time to push testing further). 

LGTM, merging.

**Pull request merged**